src/plugins/constants.ts

```diff
@@ -0,0 +1,10 @@
+import { scanPackageJson } from './scanPackageJson';
+
+export const SILO_DISCOVERY_FUNCTIONS = {
```

Comment:
probably will want to add a type here to define the interface. i.e. `SiloDiscoveryFn`

id then define the plugin resolver like:

```ts
const scanPackageJson: SiloDiscoveryFn = (arg, arg2) => {
}
```

this prevents the need to re-type the same interface every time

Reply:
bump on this, can be done later, but this will simplify things as we add more plugins

Reply:
oh yeah forgot, will add!